src/main/resources/Help/Code/Translations.fr.xml

```diff
@@ -0,0 +1,193 @@
+help.organization.title=Organisation du contenu
+help.start.description=Quelques étapes faciles pour commencer à utiliser XWiki.
+help.start.title=Pour commencer
+help.templates.article.description=Page d'article
+help.templates.article.title=Article
+help.templates.create.description=Intégrer la structure et les macros dans une page existante, et la transformer en modèle.
+help.templates.create.tip=Les {0}fournisseurs de modèles{1} présentent les modèles à la création d''une page, afin que les utilisateurs puissent les utiliser.
+help.templates.create.title=Créez votre modèle
+help.templates.description=Les modèles sont des pages prédéfinies qui sont copiées lors de la création d'une nouvelle page.
+help.templates.encyclopedia.description=Page Encyclopédie
+help.templates.encyclopedia.title=Encyclopédie
+help.templates.examples.description=Vous voulez commencer à partir d'un modèle prédéfini ? Voici quelques exemples de modèles disponibles dans XWiki
+help.templates.examples.title=Exemples
+help.templates.meeting.title=Compte-rendu de réunion
+help.templates.meeting.description=Agenda et notes
+help.templates.simple.description=Avec table des matières
+help.templates.simple.title=Page simple
+help.templates.more=Voir plus de modèles
+help.templates.title=Modèles
+help.templates.useTemplates.description=Les modèles sont des pages prédéfinies qui sont copiées lors de la création d'une nouvelle page.
+help.templates.useTemplates.tip={0}Créer des pages{1}à partir de modèles permet d''adapter rapidement un contenu prédéfini à vos besoins.
+help.templates.useTemplates.title=Utiliser les modèles
+help.title=Page d'aide
+help.title1=Comment utiliser XWiki ?
+help.title2=Principales fonctionnalités d'XWiki
+help.try.label=Essayez !
+help.videos.description=Voir nos tutoriels vidéos.
+help.videos.more=Voir plus de vidéos
+help.videos.title=Vidéos
+help.videos.videoCard1.title=Présentation de la page d'accueil
+help.videos.videoCard1.topic1=Découvrez les éléments affichés sur votre page d'accueil
+help.videos.videoCard1.topic2=Commencez à parcourir votre wiki
+help.videos.videoCard2.title=Personnalisation de la page d'accueil
+help.videos.videoCard2.topic1=Apprenez comment modifier le thème de couleur de votre wiki
+help.videos.videoCard2.topic2=Apprenez à ajouter des panels et des colonnes sur votre wiki
+help.videos.videoCard2.topic3=Apprenez comment ajouter des gadgets à votre wiki
+help.videos.videoCard3.title=Création de page et de contenu
+help.videos.videoCard3.topic1=Découvrez comment créer une page
+help.videos.videoCard3.topic2=Apprenez comment utiliser l'éditeur de texte WYSIWYG
+help.videos.videoCard3.topic3=Découvrez les modes vue et édition
+help.videos.videoCard4.title=Importer des documents Office dans XWiki
+help.videos.videoCard4.topic1=Découvrez comment importer des documents Office dans votre wiki
+help.videos.videoCard4.topic2=Apprenez à utiliser la macro "Table des Matières"'
```

Comment:
A trailing quote ' to remove at the end?

Reply:
I have removed the ' at the end.